src/cargo/core/features.rs

```diff
@@ -395,9 +394,6 @@ features! {
     // Allow to specify whether binaries should be stripped.
     (unstable, strip, "", "reference/unstable.html#profile-strip-option"),
 
-    // Specifying a minimal 'rust-version' attribute for crates
-    (unstable, rust_version, "", "reference/unstable.html#rust-version"),
```

Comment:
This should be marked as "stable", see the other entries for the format.

Reply:
Fixed.